examples/model_selection/plot_validation_curve.py

```diff
@@ -18,53 +18,24 @@
 
 from sklearn.datasets import load_digits
 from sklearn.svm import SVC
-from sklearn.model_selection import validation_curve
+from sklearn.model_selection import ValidationCurveDisplay
 
 X, y = load_digits(return_X_y=True)
 subset_mask = np.isin(y, [1, 2])  # binary classification: 1 vs 2
 X, y = X[subset_mask], y[subset_mask]
 
-param_range = np.logspace(-6, -1, 5)
-train_scores, test_scores = validation_curve(
+param_name, param_range = "gamma", np.logspace(-6, -1, 5)
```

Comment:
No need to define local variables anymore. We can directly pass those to `from_estimator` as kwargs.

```suggestion
```